backend/fregepoc/repositories/models.py

```diff
@@ -79,6 +79,12 @@ class RepositoryFile(models.Model):
         null=True,
         help_text=_("File path, relative to the repository root."),
     )
+    absolute_file_path = models.CharField(  # noqa: DJ01
+        max_length=512,
+        blank=True,
+        null=True,
+        help_text=_("Absolute file path"),
+    )
```

Comment:
It's only being used in `analyze_file_task` to delete the file, I think it can just be replaced with something like 
```python
absolute_file_path = get_repo_local_path(repo_file.repository) / repo_file.relative_file_path
```